ports/qemu/mcu/arm/startup.c

```diff
@@ -27,6 +27,7 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdnoreturn.h>
```

Comment:
Is this header C99?

Can we instead just use NORETURN which is already defined by us?

Reply:
I thought this was C99 but turns out it's C11.  I recall I looked for `MP_NORETURN` but found nothing, so I guess that's an exception to the naming convention.

Still, thanks for noticing, should be addressed now.

Reply:
I also first grep'd for `MP_NORETURN` before realising it's `NORETURN`.

It would probably be good for us to rename it to `MP_NORETURN`.

Reply:
> It would probably be good for us to rename it to `MP_NORETURN`.

I came up with https://github.com/agatti/micropython/commit/981f81e0e1d913bf08f40c2f970b2f70b554f697 but I'll wait for this PR to be merged and update my commit before creating a new PR for the macro rename.